### ISSUE TITLE
More logs in coordinator

### ISSUE THREAD

**Issue by bonomat** [get10101/10101#507]
Our coordinator logs are not that helpful at the moment. This is not ideal because it makes it hard to follow-up what is happening. 

A few things we can do to improve this (open for debate and not complete)
- [x] JSON output (makes it easier to parse in grafana)
- [ ] Distinguish between an app user and another node (not sure if this is possible)
- [ ] Log channel stats, e.g. how much money do we have on either side of the channel
- [x] Log new order/position details (e.g. amounts, expiry, leverages, peer_id, our_margin, their_margin, timestamp)
- [x] Log close order/position details (e.g. amounts, expiry, leverages, peer_id, our_margin, their_margin, timestamp)
- [x] Log if a payment goes through us
-...

**Comment by klochowicz** [comment]
#511 - Json logging
#512 - logs about the position

Will open another one with the lightning stats, that one won't be as fast - just churning it in parts to increase visibility as we go.

**Comment by klochowicz** [comment]
re: "log if a payment goes through us" - I've added logs for all the missing LDK events, hopefully we'll see that in #513

**Comment by klochowicz** [comment]
we also were unnecessarily abstracting away error details from the bdk wallet: https://github.com/get10101/10101/pull/526

**Comment by bonomat** [comment]
> re: "log if a payment goes through us" - I've added logs for all the missing LDK events, hopefully we'll see that in #513
> 
> crossed it from the list, but let me know if the scope was bigger than that @bonomat

Nice, let's hope this will give us some more clarity :)

**Comment by bonomat** [comment]
@klochowicz : are you working on the last two points as well?

**Comment by klochowicz** [comment]
> @klochowicz : are you working on the last two points as well?

not at the moment, as I'm dealing with the critical production bug - I'll unassign myself for now.
